.github/workflows/build.yaml

```diff
@@ -0,0 +1,19 @@
+name: Build package
```

Comment:
gonna do this over in this pr instead -- need to fix the pnpm install step so typechecking works in ci

https://github.com/OpenRouterTeam/ai-sdk-provider/pull/30

Reply:
cool beans!